site/docs/encoding/scale.md

```diff
@@ -458,6 +458,13 @@ To provide themes for all scales, the scale config (`config: {scale: {...}}`) ca
 
 {% include table.html props="invalid" source="ScaleConfig" %}
 
+#### Animation
+
+<!-- TODO(jzong): uncomment animationDuration when linear scales for animation are implemented -->
```

Comment:
Please reference an issue so we know when this work is done and keep track of what needs to be done here.